config/database.js

```diff
@@ -1,10 +1,8 @@
-const host = process.env.DB_HOST || 'localhost';
-const port = process.env.DB_PORT || '27017';
-const database = process.env.DB_NAME || 'reader-api-dev';
+require('dotenv').config();
```

Comment:
I don't know why the config folder don't understand env variables. although the index.js file declared require('dotenv').config() before calling the config file.